### DISCUSSION TITLE
AMD graphic card

### DISCUSSION THREAD

**Discussion by luanwma** [alicevision/meshroom-manual#39] [General]
Hey my AMD graphic card is with the newest version of driver but even that way is not possible to complete a fully reconstruction, am I doing something wrong or the meshroom does not support AMD graphics? 
![error](https://user-images.githubusercontent.com/20958996/175037765-daa42a1d-1f7c-445c-a078-0acefa3f8136.PNG)

**Comment by simogasp** [comment]
It does not support AMD cards, only NVIDIA ones.

**Reply by luanwma** [reply]
okay thanks. 
if I could suggest something, that would be the only suggestion.
but the meshroom is definitely one of the best reconstruction software I've ever used!
